<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatDashboard</class>
 <widget class="QDialog" name="ChatDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:color rgb(53, 132, 228);
border: 2px solid rgb(53, 132, 228);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:color rgb(26, 95, 180);
border-radius: 15px;
border: 2px solid rgb(53, 132, 228);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>25</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>                                        Chatter Box</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:color rgb(53, 132, 228);
border: 2px solid rgb(53, 132, 228);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="btnSignOut">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background:color #1AC0C6;
border-radius: 15px;
  border: 2px solid;
  padding: 10px;
  width: 200px;
  height: 150px;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/resources/images/log-out.svg</normaloff>:/resources/images/log-out.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Hello</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblUserName">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>15</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnEditProfle">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background:color #1AC0C6;
border-radius: 15px;
  border: 2px solid;
  padding: 10px;

QPushButton#btnEditProfile {
    background-color: yellow;
};

QPushButton#btnEditProfile:hover {
    background-color: rgb(224, 255, 0);
};

QPushButton#btnEditProfile:pressed {
    background-color: rgb(224, 0, 0);     
}</string>
              </property>
              <property name="text">
               <string>Edit Profile</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnChatRooms">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background:color #1AC0C6;
border-radius: 15px;
  border: 2px solid;
  padding: 10px;
  width: 200px;
  height: 150px;</string>
              </property>
              <property name="text">
               <string>Chat Rooms</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnSaveChatHistory">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background:color #1AC0C6;
border-radius: 15px;
  border: 2px solid;
  padding: 10px;
  width: 200px;
  height: 150px;</string>
              </property>
              <property name="text">
               <string>Save Chat History</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnViewChatHistory">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background:color #1AC0C6;
border-radius: 15px;
  border: 2px solid;
  padding: 10px;
  width: 200px;
  height: 150px;</string>
              </property>
              <property name="text">
               <string>View Chat History</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="styleSheet">
            <string notr="true">background:color rgb(53, 132, 228);
border: 2px solid rgb(53, 132, 228);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QListWidget" name="listWidgetMessages">
              <property name="styleSheet">
               <string notr="true">background:color #fff;
border-radius: 15px;
border: 2px solid #fff;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_7">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:color rgb(53, 132, 228);
border: 2px solid rgb(53, 132, 228);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">font:colour rgb(243, 243, 243)
</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resources.qrc">:/resources/images/user.svg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">font:colour rgb(243, 243, 243)
</string>
              </property>
              <property name="text">
               <string>Online</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listWidgetOnline">
              <property name="styleSheet">
               <string notr="true">background:color #fff;
border-radius: 15px;
border: 2px solid #fff;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">font:colour rgb(243, 243, 243)
</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resources.qrc">:/resources/images/message-square.svg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">font:colour rgb(243, 243, 243)
</string>
              </property>
              <property name="text">
               <string>Group Chats</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listWidgetGroups">
              <property name="styleSheet">
               <string notr="true">background:color #fff;
border-radius: 15px;
border: 2px solid #fff;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">font:colour rgb(243, 243, 243)
</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resources.qrc">:/resources/images/book-open.svg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">font:colour rgb(243, 243, 243)
</string>
              </property>
              <property name="text">
               <string>Personal Contacts</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listWidgetContacts">
              <property name="styleSheet">
               <string notr="true">background:color #fff;
border-radius: 15px;
border: 2px solid #fff;
</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:color rgb(26, 95, 180);
border-radius: 15px;
border: 2px solid rgb(53, 132, 228);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="txtMessage">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:color #fff;
border-radius: 10px;
  border: 2px solid #fff;</string>
           </property>
           <property name="placeholderText">
            <string>Type message here...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sendButton">
           <property name="styleSheet">
            <string notr="true">
  border:none;
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/resources/images/navigation.svg</normaloff>:/resources/images/navigation.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>42</width>
             <height>42</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
